cmd/globalAlignment/testdata/faOut_test.fa

```diff
@@ -0,0 +1,4 @@
+>chelsea
+TTGTTATTC
+>eric
+TTG---TTC
```

Comment:
It looks like you may need an end-of-line character here

Reply:
Yes! I just rerun the program to update this output file with \n. Thank you!